erpnext/accounts/doctype/pos_profile/pos_profile.json

```diff
@@ -306,10 +307,17 @@
    "fieldname": "ignore_pricing_rule",
    "fieldtype": "Check",
    "label": "Ignore Pricing Rule"
+  },
+  {
+   "default": "1",
+   "fieldname": "show_only_available_items",
+   "fieldtype": "Check",
+   "label": "Show only available items"
   }
  ],
  "icon": "icon-cog",
  "idx": 1,
+ "index_web_pages_for_search": 1,
  "links": [],
  "modified": "2020-10-01 17:29:27.759088",
```

Comment:
As modified date is not updated getting below error while creating the POS Opening Entry

<img width="793" alt="Screenshot 2020-10-13 at 11 59 58 PM" src="https://user-images.githubusercontent.com/8780500/95901173-9202e300-0db0-11eb-81ae-a2c0036d3bf8.png">

<img width="754" alt="Screenshot 2020-10-13 at 11 59 52 PM" src="https://user-images.githubusercontent.com/8780500/95901195-9929f100-0db0-11eb-9756-23f2305e2958.png">